docs/application/vscode-ext/web.md

```diff
@@ -406,6 +377,93 @@ VS Code Extension for Tizen Web supports intelliSense. You can use these feature
 
   ![Document Viewer](media/tizen_web23.gif)
 
+## Tizen Web with Design Editor
+
+Visual Studio Code Extension for Tizen Web that supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** and **Redo**.
+You must select the **HTML extension file** in the file explorer before you launch Design Editor.
```

Comment:
@singa2000 
Suggestion:
How about adding a line space here? By giving a line space the prerequisite will be slightly highlighted.
Please verify.  If you feel it is invalid, kindly ignore this comment.

Reply:
Also, the "that" before the "supports" should be removed as https://github.com/Samsung/tizen-docs/pull/1016#discussion_r375615982.

Visual Studio Code Extension for Tizen Web supports ...

Reply:
done

Reply:
Just for a reference. To insert a empty line, we need 2 lines in md file.
Only the 1 line doesn't create a new line.

Below is the result of this md file.

![image](https://user-images.githubusercontent.com/9958374/73996896-52be0000-49a0-11ea-9dce-9e33bfb59f9a.png)